wizard/cypress/e2e/uploadCsv.cy.js

```diff
@@ -29,7 +34,34 @@ describe('Wizard from local CSV', () => {
       })
     })
     cy.get('.uploadedFileData').contains('base.csv')
-    cy.get('[data-testid="stMarkdownContainer"]').contains('Submit').click()
-    cy.get('[data-testid="stJson"]').should('contain', '@context')
+    cy.get('button').contains('Add').click()
+  })
+  it('should allow uploading existing croissant files', () => {
+    const resizeObserverLoopErrRe = /ResizeObserver loop limit exceeded/
+
+    // consensus was that this exception didn't matter mostly, and was intermittent when running tests.
```

Comment:
This code is repeated. From the official Cypress example, the good practice seems to be the following structure ([source](https://github.com/cypress-io/cypress-example-kitchensink/tree/master/cypress)):

```
|_ cypress
  |_ e2e
  |_ fixtures
  |_ support
    |_ resize-observer.js
    |_ e2e.js
```

Where `resizeOberver.js` would have the code and `e2e.js` imports it.